flexget/ui/v1/__init__.py

```diff
@@ -5,7 +5,7 @@
 import os
 import fnmatch
 
-from flask import send_from_directory, Flask
+from flask import send_from_directory, Flask, abort
```

Comment:
why add `abort`?

Reply:
Ah, i was using it to send a 404 but figured out a different way to handle running from github. Removed it.